.ci/cico_build_deploy_test_rhche.sh

```diff
@@ -56,7 +56,7 @@ else
   echo "Switching to project failed, probably not exists [$?]. Creating..."
   oc new-project "${PROJECT_NAMESPACE}" --display-name="PR ${RH_PULL_REQUEST_ID} - Automated Deployment" > /dev/null 2>&1
 fi
-oc policy add-role-to-user edit Katka92 ScrewTSW rhopp tomgeorge ibuziuk amisevsk davidfestal skabashnyuk -n $PROJECT_NAMESPACE
+oc policy add-role-to-user edit ibuziuk -n $PROJECT_NAMESPACE
```

Comment:
One thing I don't understand is why are we being removed from the adminlist?

Reply:
because I want to test this first with the minimal users, and othes would be added in a separate PR

Reply:
👍